### DISCUSSION TITLE
Limit connection by ip address?

### DISCUSSION THREAD

**Discussion by ahhmino** [supabase/supabase#1585] [Questions] [answered]
I'd like to set up an api server in front of the postgres database, and I'm wondering is there any way to limit connections by incoming ip address, so I could restrict connections to be only allowed from a particular server? Thanks

**Comment by steve-chavez** [comment, marked as answer]
> Limit connection by ip address?

This is not user-configurable right now, but it will be once we update Kong to use a db-based configuration(soon). For more details, see this discussion: https://github.com/supabase/supabase/discussions/1524#discussioncomment-746106
